arm-appinsights/2015-05-01/swagger/syntheticMonitor_API.json

```diff
@@ -0,0 +1,402 @@
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/webtests": {
+      "get": {
+        "description": "Get all Application Insights synthetic monitors defined within a specified resource group.",
+        "operationId": "SyntheticMonitor_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list containing 0 or more synthetic monitor definitions.",
+            "schema": {
+              "$ref": "#/definitions/SyntheticMonitorListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SyntheticMonitorListByResourceGroup": {
+            "$ref": "../examples/SyntheticMonitorListByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/webtests/{webTestName}": {
+      "get": {
+        "description": "Get a specific Application Insights synthetic monitor definition.",
+        "operationId": "SyntheticMonitor_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/WebTestNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ann Application Insights synthetic monitor definition.",
+            "schema": {
+              "$ref": "#/definitions/SyntheticMonitor"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SyntheticMonitorGet": {
+            "$ref": "../examples/SyntheticMonitorGet.json"
+          }
+        }
+      },
+      "put": {
+        "description": "Creates or updates an Application Insights synthetic monitor definition.",
+        "operationId": "SyntheticMonitor_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/WebTestNameParameter"
+          },
+          {
+            "name": "WebTestDefinition",
```

Comment:
- [x] **Done** as stated above, I've wholesale renamed the Synthetic Monitor API back to WebTests. (Including the file name!)